WtWSMS/common/on_action/sea_minority_game_start.txt

```diff
@@ -0,0 +1,1572 @@
+						this = title:c_khumar
+					}
+				}
+				add_small_culture_to_county_effect = { CULTURE = culture:bosporan }
+			}
+			##Helleno-Thracians
+			every_county = {
+				limit = {
+					OR = {
+						this = title:c_aytos
+						this = title:c_mesembria
+						this = title:c_constanta
+					}
+				}
+				add_large_culture_to_county_effect = { CULTURE = culture:helleno_thracian }
+			}
+			#Levantine Heritage
+			##Arameans
+			add_culture_to_region_effect = {
+				REGIONAL_TRIGGER = aramean_location_trigger
+				CULTURE = culture:aramean
+			}
+			##Assyrians
+			add_culture_to_region_effect = {
+				REGIONAL_TRIGGER = assyrian_location_trigger
+				CULTURE = culture:assyrian
+			}
+			#Arabic Heritage
+			##Khalans
+			every_county = {
+				limit = {
+					OR = {
+						#Ghassanids
+						this = title:c_hauran
+						this = title:c_balqa
+						this = title:c_irbid
+						this = title:c_damascus
+						#Lakhmids
+						this = title:c_waqisa
+						this = title:c_thalabiya
+						this = title:c_akhadid
+						this = title:c_karbala
+					}
+				}
+				add_small_culture_to_county_effect = { CULTURE = culture:khalan }
+			}
+			#Caucasian Heritage
+			##Aghwank
+			every_county = {
+				limit = {
+					OR = {
+						this = title:c_shirvan
+						this = title:c_mugan
+					}
+				}
+				add_large_culture_to_county_effect = { CULTURE = culture:caucasian_albanian }
+			}
+			every_county = {
+				limit = {
+					OR = {
+						this = title:c_odishi
+						this = title:c_guria
+						this = title:c_acampse
+					}
+				}
+				add_small_culture_to_county_effect = { CULTURE = culture:georgian }
+			}
+			
+			## Other Special Regions
+			#========================
+			### Romano-Gaul presence throughout former Gaul
+			add_culture_to_region_granular_effect = {
+				REGIONAL_TRIGGER = e_gaul_romance_trigger
+				CULTURE = culture:romano_gallic
+			}
+			add_faith_to_region_granular_effect = {
+				REGIONAL_TRIGGER = e_gaul_nicene_trigger
+				FAITH = faith:gallican_church
+			}
+			### North Africa
+			#Western North Africa should have populations of Romano-African African Rite Nicenes
+			every_county = {
+				limit = {
+					this.duchy = title:d_caesariensis
+					is_coastal_county = yes
+				}
+				add_large_culture_and_faith_to_county_effect = {
+					FAITH = faith:african_church
+					CULTURE = culture:romano_african
+				}
+			}
+			every_county = {
+				limit = {
+					OR = {
+						this = title:c_melilla
+						this = title:c_gharb
+						this = title:c_tlemcen
+					}
+				}
+				add_small_culture_and_faith_to_county_effect = {
+					FAITH = faith:african_church
+					CULTURE = culture:romano_african
+				}
+			}
+			#Libyan Gurzils
+			every_county = {
+				limit = {
+					OR = {
+						this.duchy = title:d_leptis_magna
+						this = title:c_tripolitana
+					}
+				}
+				add_small_culture_and_faith_to_county_effect = {
+					FAITH = faith:gurzil
+					CULTURE = culture:libyan
+				}
+			}
+			### Vandal Settlement in North Africa & Hispaniae
```

Comment:
From what I see there are no longer any Vandal minorities in Hispania. This should probably be the case since the sources lead us to believe that Vandals remained in significant numbers in the south of the peninsula (roughly the counties used in CK2, I believe). At the very least significant enough to constitute a military force.